templates/web/bromley/around/_report_banner.html

```diff
@@ -0,0 +1,10 @@
+<h1 class="big-green-banner">
```

Comment:
This file means the bromley check in `templates/web/fixmystreet/around/_report_banner.html` can be removed.